### PR TITLE
[hi] Added content/hi/docs/concepts/architecture/garbage-collection.md

### PR DESCRIPTION
**Pull request by akash-kumar-saw**
Issue Number #39548

Localized en/docs/concepts/architecture/garbage-collection.md in hindi language :)

### CODE REVIEW

File: content/hi/docs/concepts/architecture/garbage-collection.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+[कॉन्फ़िगरेशन फ़ाइल](/docs/tasks/administer-cluster/kubelet-config-file/) का उपयोग करके क्यूबलेट
+और कचरा संग्रहण से संबंधित मापदंडों को उपयोग करके बदलें
+[`KubletConfiguration`](/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubletConfiguration)
+resource type.
+
+### कंटेनर छवि जीवनचक्र
+
+कुबेरनेट्स अपने *छवि प्रबंधक* के माध्यम से सभी छवियों के जीवनचक्र का प्रबंधन करता है,
+जो क्यूबलेट का हिस्सा है, के सहयोग से
+{{< glosary_tooltip text="cadvisor" term_id="cadvisor" >}}. क्यूबलेट
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/garbage-collection.md:128]: Typo in the `glossary_tooltip` shortcode name. Please consider localizing the "text" parameter passed to the shortcode to Hindi.

File: content/hi/docs/concepts/architecture/garbage-collection.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+क्लस्टर-दायरे वाले आश्रित केवल क्लस्टर-दायरे वाले स्वामियों को निर्दिष्ट कर सकते हैं।
+v1.20+ में, यदि एक क्लस्टर-दायरे वाला निर्भर एक स्वामी के रूप में एक नामस्थान प्रकार निर्दिष्ट करता है,
+इसे एक अनसुलझे मालिक संदर्भ के रूप में माना जाता है, और यह कचरा एकत्र करने में सक्षम नहीं है।
+
+v1.20+ में, अगर गारबेज संग्राहक एक अमान्य क्रॉस-नेमस्पेस `ओनर रेफरेंस` का पता लगाता है,
+या एक क्लस्टर-स्कोप्ड निर्भर एक `ओनर रेफरेंस` के साथ एक नेमस्पेस्ड प्रकार, एक चेतावनी ईवेंट को संदर्भित करता है
+'OwnerRefInvalidNamespace' के कारण और अमान्य आश्रित के 'शामिल ऑब्जेक्ट' की सूचना दी गई है।
+आप उस तरह के इवेंट को चलाकर चेक कर सकते हैं
+`kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace`।
+{{< /नोट >}}
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/garbage-collection.md:57]: Avoid localizing the shortcode name to Hindi, as it results in [errors](https://app.netlify.com/sites/kubernetes-io-main-staging/deploys/65723d14589bf8000808beb6). Keep the shortcode name as it is and localize the content inside.

```suggestion
{{< note >}}
क्रॉस-नेमस्पेस स्वामी संदर्भ डिज़ाइन द्वारा अस्वीकृत हैं।
नेमस्पेस्ड आश्रित क्लस्टर-स्कोप्ड या नेमस्पेस्ड मालिकों को निर्दिष्ट कर सकते हैं।
नामस्थान स्वामी **आवश्यक** उसी नामस्थान में आश्रित के रूप में मौजूद होना चाहिए।
यदि ऐसा नहीं होता है, तो स्वामी संदर्भ को अनुपस्थित और आश्रित के रूप में माना जाता है
एक बार सभी स्वामियों के अनुपस्थित होने की पुष्टि हो जाने के बाद उसे हटाया जा सकता है।

क्लस्टर-दायरे वाले आश्रित केवल क्लस्टर-दायरे वाले स्वामियों को निर्दिष्ट कर सकते हैं।
v1.20+ में, यदि एक क्लस्टर-दायरे वाला निर्भर एक स्वामी के रूप में एक नामस्थान प्रकार निर्दिष्ट करता है,
इसे एक अनसुलझे मालिक संदर्भ के रूप में माना जाता है, और यह कचरा एकत्र करने में सक्षम नहीं है।

v1.20+ में, अगर गारबेज संग्राहक एक अमान्य क्रॉस-नेमस्पेस `ओनर रेफरेंस` का पता लगाता है,
या एक क्लस्टर-स्कोप्ड निर्भर एक `ओनर रेफरेंस` के साथ एक नेमस्पेस्ड प्रकार, एक चेतावनी ईवेंट को संदर्भित करता है
'OwnerRefInvalidNamespace' के कारण और अमान्य आश्रित के 'शामिल ऑब्जेक्ट' की सूचना दी गई है।
आप उस तरह के इवेंट को चलाकर चेक कर सकते हैं
`kubectl get events -A --field-selector=reason=OwnerRefInvalidNamespace`।
{{< /note >}}
```

**Comment by utkarsh-singh1** [comment]
@akash-kumar-saw , Thanks for the translation , but Note these points before push your next PR-

- Check output from english to hindi translator tools as most of the tools can provide translation but sometimes they can not provide right phrases of the paragraph or sentences, words may be misplaced many times .

- Don't translate values inside these tags (<>).

- Before you submit another commit, take a minute to inspect/review the changes specially for localisation.

**Comment by utkarsh-singh1** [comment]
Hi @akash-kumar-saw, Before you change term_id under glossary_definition, check [here](https://kubernetes.io/hi/docs/reference/glossary/?fundamental=true), if it's term_id is available in English use it in the original format.

File: content/hi/docs/concepts/architecture/garbage-collection.md [hunk 3]
```diff
@@ -0,0 +1,183 @@
+---
+title: कचरा संग्रहण
+content_type: concept
+weight: 70
+---
+
+<!-- overview -->
+{{<glossary_definition term_id="गार्बेज-कलेक्शन" length="short">}} यह
```
Comment (dipesh-rawat) [inline content/hi/docs/concepts/architecture/garbage-collection.md:8]: @akash-kumar-saw For `glossary_definition` to work as expected we should localize the glossary entry for "garbage-collection" (i.e. [en/docs/reference/glossary/garbage-collection.md](https://github.com/kubernetes/website/blob/main/content/en/docs/reference/glossary/garbage-collection.md)) and incorporate it into this pull request by adding to list of Hindi glossary items inside [/hi/docs/reference/glossary](https://github.com/kubernetes/website/tree/main/content/hi/docs/reference/glossary). Also, please ensure that `term_id` is in English.

```suggestion
{{<glossary_definition term_id="garbage-collection" length="short">}} यह
```

**Comment by utkarsh-singh1** [comment]
Hi @akash-kumar-saw , avilable to change the review by @dipesh-rawat ?

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
@akash-kumar-saw : Kindly read the machine translation and see if it makes sense to you, as a Hindi reader, before copy-pasting it and submitting a PR.

Directly copy-pasting and requesting a review is not helpful to us, as reviewers and approvers, and to you, as a contributor.

Currently, every line is directly copy-pasted and needs extensive rewriting because

- the grammar is wrong,
- the conventions haven't been adhered to, and
- words that shouldn't have been translated have become entire phrases.

**Pull request closed without merging**